packages/compass-collection/src/components/collection-stats/collection-stats.tsx

```diff
@@ -31,24 +31,61 @@ const tooltipIndexeListStyles = css({
 });
 
 type CollectionStatsProps = {
-  documentCount: string;
-  storageSize: string;
-  avgDocumentSize: string;
-  indexCount: string;
-  totalIndexSize: string;
-  avgIndexSize: string;
+  stats: CollectionState['stats'];
   isTimeSeries?: boolean;
 };
 
+const INVALID = 'N/A';
+
+const avg = (size: number, count: number) => {
+  if (count <= 0) {
+    return 0;
+  }
+  return size / count;
+};
+
+const isNumber = (val: any): val is number => {
+  return typeof val === 'number' && !isNaN(val);
+};
+
+const format = (value: any, format = 'a') => {
```

Comment:
This is just old logic that was part of the stats store that was moved to presentation layer that is a better fit for it